agent/tool-scripts/base-tool

```diff
@@ -191,9 +191,13 @@ esac
 
 function usage {
 	printf -- "The following options are available:\n\n"
+	printf -- "Operational options - do *not* specify any of these when you register a tool:\n\n"
 	printf -- "\t--install                 verify this perf tool will work\n\n"
 	printf -- "\t--start|stop|postprocess  start/stop/postprocess the data collection\n"
 	printf -- "\t-d str|--dir=str          directory to store data collection (required)\n"
+
+        printf -- "\nTool-specific options - these should be specified when you register a tool:\n\n"
+
 	case "${tool}" in
 	blktrace|bpftrace|kvmtrace|lockstat|oc|perf|strace|systemtap|tcpdump)
 		:  # These tools don't use interval
```

Comment:
Given that we now have "headers" at lines 199 and 286, we should probably print out "none" or something here.  (See the `lockstat` gold file....)

Reply:
Either that or defer the header until we know there's something to print ... but a "none" would be easier to implement. 😆